src/content/docs/iast/troubleshoot-iast.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshooting for interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+  - Troubleshooting
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
+## I don't see my application in New Relic
+
+Confirm that the application has started and then review the application's logs for further information.
+
+## How do I make sure that the security agent is working?
+
+In the application's `stdout` you will see this message:
+
+```shell
+This application instance is now being scanned by New Relic Security under id {{UUID}}
+```
+
+In the `nr-security-home/logs` directory, search for the file called `java-security-collector-init.log`, replace `java` in the path depending on language you used, and make sure these steps work for you:
+
+1. The security agent is starting.
+2. It generates a unique identifier. For web socket connection, you'll see Node auth headers.
```

Comment:
Where do I find this unique ID?

Reply:
Where will I see the Node auth headers? Is this all self-evident?

Reply:
I'll ask the dev about this ID.